go-controller/pkg/cluster/node.go

```diff
@@ -76,17 +76,25 @@ func (cluster *OvnClusterController) StartClusterNode(name string) error {
 	var cidr string
 	var connected bool
 
+	var wg sync.WaitGroup
+	type waitResult struct {
+		Error error
+	}
+
 	for _, clusterSubnet := range config.Default.ClusterSubnets {
 		clusterSubnets = append(clusterSubnets, clusterSubnet.CIDR.String())
 	}
 
+	node, err = cluster.Kube.GetNode(name)
+	if err != nil {
+		logrus.Errorf("Error starting node %s, no node found - %v", name, err)
+		return err
+	}
+
+	messages := make(chan waitResult)
```

Comment:
@mccv1r0 so, turns out we actually don't need a structure to wrap the error. We can just use a bare error, and Go will happily accept a `nil` in the channel. So let's lose the waitResult wrapper.